src/launcher/java/org/truffleruby/launcher/options/OptionsCatalog.java

```diff
@@ -611,6 +615,8 @@
                 return INTEROP_WRITE_CACHE;
             case "ruby.interop.invoke.cache":
                 return INTEROP_INVOKE_CACHE;
+            case "ruby.interop.new.cache":
+                return INTEROP_NEW_CACHE;
```

Comment:
I don't think there is much value to have a different cache per operation, why not just a single INTEROP_CACHE?